packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -266,7 +278,8 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     return Object.assign({}, styles?.sendBox);
   }, [styles?.sendBox]);
 
-  const userId = toFlatCommunicationIdentifier(adapter.getState().userId);
+  const userIdObject = useSelector(getUserId);
+  const userId = toFlatCommunicationIdentifier(userIdObject);
```

Comment:
it's probably a good idea to use `useMemo` here

Reply:
As it's flattened to a string, strings typically don't need memoized (as they equate equally when compared unlike objects)

Reply:
Good to know!